.changelog/template.md.j2

```diff
@@ -0,0 +1,44 @@
+{# This a Jinja2 template for preparing a new section of the Change Log with the towncrier utility.#}
+## {{ versiondata.version }} ({{ versiondata.date }})
+
+| Name         | Version   |
+|:-------------|:---------:|
+| Rosetta API  | <VERSION> |
+| Oasis Core   | <VERSION> |
```

Comment:
I would omit this part for now. Later, we could introduce it to indicate which Oasis Indexer version is compatible with the current Oasis Explorer release.